src/numJewelsInStones.js

```diff
@@ -28,7 +28,28 @@
  * @return {number}
  */
 function numJewelsInStones(jewels, stones) {
-  // write code here
+  let countJewels = 0;
+  const regExpNotWords = /[^\w]|\d/g;
```

Comment:
```suggestion
  const pattern = /[^\w]|\d/g;
```
когда он один, можно не усложнять названия переменных

если ты хочешь оставить только буквы, то:
```suggestion
  const pattern = /[^a-z]/gi;
```